### ISSUE TITLE
WDC index

### ISSUE THREAD

**Issue by ran-yu** [Timpy/Glimmer#3]
Hi,

I tried to build the index, but the cluster I use doesn't have enough memory for building index from the WDC dataset. So I want to know if the index from Yahoo can be downloaded? That would be really helpful.

Another question: is the url/ domain information of the tuples saved in the index?

Thanks a lot!!

**Comment by DengueTim** [comment]
Hi Ran-yu,
   Unfortunately I'm not working at Yahoo anymore and I don't know if this
project is still being actively maintained by Yahoo.   I have sent a
message to the team at Yahoo about making the WDC indexes(form
glimmer.research.yahoo.com) available to download via the web.   With a bit
of luck they will find time to make the configuration changes needed.

Regards,
  Tim.

**Comment by ran-yu** [comment]
Hi Tim,

Thanks a lot. Is there anyone I can contact about this project?

Cheers,

Ran Yu
Mail: ranyu.zh@gmail.com
Cellphone: +49 176 6701 7958

**Comment by DengueTim** [comment]
I think you are already in contact with Peter Mika right?

**Comment by ran-yu** [comment]
Yes, I did, I was not sure if he is the right contact. Thanks :)

Ran Yu
Mail: ranyu.zh@gmail.com
Cellphone: +49 176 6701 7958
